value.go

```diff
@@ -43,7 +43,7 @@ var (
 
 // ToValue will convert an interface{} value to a value digestible by otto/JavaScript
 //
-// This function will not work for advanced types (struct, map, slice/array, etc.) and
+// This function will not work for advanced types (struct, map, slice/array, func, etc.) and
```

Comment:
This seems to be in conflict to the PR description, did I miss-interpret your subject / message or has this been change been reversed some how?